charts/redpanda/templates/_helpers.tpl

```diff
@@ -304,6 +304,21 @@ Generate configuration needed for rpk
 {{- dig "mechanism" (include "sasl-mechanism" $) $.user -}}
 {{- end -}}
 
+{{- define "sasl-first-users" -}}
+{{- toJson ( dict "list" .Values.auth.sasl.users ) -}}
```

Comment:
I don't understand how this can be `first` sasl user template. If I read this correctly, whole users list will be marshalled to json, right?

Reply:
The first would presumably be the admin user, this is the previous way this was "working" which i would argue never really did. Regardless I need one username/pass combination to pass into flags, a choice has to be made.

Reply:
Then I would rename it to `sasl-users`